ga4gh/datamodel/reads.py

```diff
@@ -68,6 +68,8 @@ class SamFlags(object):
     """
     NUMBER_READS = 0x1
     PROPER_PLACEMENT = 0x2
+    UNMAPPED = 0x4
+    NEXT_SEGMENT_UNMAPPED = 0x8
```

Comment:
I'm out of my depth here I'm afraid @dcolligan, I have nothing of any value to add.

The thing I can suggest is to ping @calbach, he's knows the reads API as well as anyone, and might be able to point us at what Google does in this case.